CONTRIBUTING.md

```diff
@@ -202,16 +202,29 @@ The difference between `EarlyLintPass` and `LateLintPass` is that the methods of
 
 That's why the `else_if_without_else` example uses the `register_early_lint_pass` function. Because the [actual lint logic][else_if_without_else] does not depend on any type information.
 
-### Fixing nightly build failures
+### Fixing build failures caused by Rust
 
-Clippy will sometimes break with new nightly version releases. This is expected because Clippy still depends on nightly Rust. Most of the times we have to adapt to the changes and only very rarely there's an actual bug in rust.
+Clippy will sometimes break because Clippy still depends on unstable internal Rust features. Most of the times we have to adapt to the changes and only very rarely there's an actual bug in rust.
```

Comment:
* Could we avoid repeating `Clippy` within one sentence?
* `s/rust/Rust/`

Reply:
done